src/barfTurn.ts

```diff
@@ -286,6 +288,19 @@ const turns: AdventureAction[] = [
     spendsTurn: false,
     sobriety: Sobriety.EITHER,
   },
+  {
+    name: "Heavy Rains Rain Main",
+    available: () => have($skill`Rain Man`) && myRain() >= 50,
```

Comment:
This doesn't check whether you have embezzler as rainmannable--what are the conditions for that? I know that having it in your manuel is helpful, but I assume that rain man isn't wholly gated behind manuel?

Reply:
have it in your manuel, or have fought it in the current HR ascension as I understand it

Reply:
yes that is correct, if you don't have manuel you can hit the choice adventure once to get the full list for free if you want to know if you can summon something. Or just only support manuel havers.

Reply:
this is the check from autoscend if you wanted to verify first
```
boolean canRainManSummon(monster target)
{
	if(!have_skill($skill[Rain Man]) || my_rain() < 50)
	{
		return false;
	}

	// Can only rain man summon copyable monsters
	if(!target.copyable || target.id < 0)
	{
		return false;
	}

	// Can summon any monster with available factoids
	if (target.monster_factoids_available(false) > 0)
	{
		return true;
	}

	// Check the page text
	auto_log_info(target + " factoids unavailable, checking Rain Man if summon is possible", "blue");
	buffer page = visit_url("runskillz.php?pwd&action=Skillz&whichskill=16011&quantity=1");
	// Escape
	run_choice(2);

	return page.contains_text("<option value=" + target.id + ">");
}
```